.github/workflows/pr_check.yml

```diff
@@ -11,6 +11,11 @@ env:
   TEST_SSL_KEY: ${{ secrets.TEST_SSL_KEY }}
 
 jobs:
+  semgrep:
```

Comment:
To make the new job more consistency with others, you should update the following list below.
1. rename job property `semgrep` to scan-semgrep`
2. rename job `scan` to `scan-sonarcloud`
3. add property `name: Scan / Semgrep`
4. move `scan-semgrep` job after the `scan-sonarcloud` job.

Reply:
done